src/SonarScanner.MSBuild/ArgumentProcessor.cs

```diff
@@ -18,12 +18,6 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-using System;
```

Comment:
this file is just irrelevant cleanup from me checking where makes sense to put the telemetry message.
I left it as it's a very simple PR so it does not really add noise.